django/contrib/gis/db/backends/spatialite/operations.py

```diff
@@ -159,10 +159,14 @@ def geos_version(self):
         "Return the version of GEOS used by SpatiaLite as a string."
         return self._get_spatialite_func('geos_version()')
 
-    def proj4_version(self):
-        "Return the version of the PROJ.4 library used by SpatiaLite."
+    def proj_version(self):
+        "Return the version of the PROJ library used by SpatiaLite."
         return self._get_spatialite_func('proj4_version()')
 
+    def proj4_version(self):
+        "Alias for proj_version."
+        return self.proj_version()
+
```

Comment:
```python
    proj4_version = proj_version  # Alias for backward compatibility.
```

Although, do we even need this? We normally accept documented backward incompatible changes in database backends.

Reply:
This method is specific for the SpatiaLite backend. Moreover it's unused and undocumented since its introduction in ff60c5f9de3e8690d1e86f3e9e3f7248a15397c8. I will remove an alias and add small note (after #12854).